docs/user_documentation/guide-admin.md

```diff
@@ -75,6 +75,22 @@ If at any time you want to edit an existing user, you can click the icon in the
 
 Managing your users is quick and easy. And opens up way to our next big security item, setting permissions for your data.
 
+# Authentication
+In MOLGENIS you can configure two authentication methods:
+ * Default MOLGENIS-authentication
+ * Google-Authentication
+ 
+**Default-authentication**  
+In the App Settings:
+ * Set user-moderation to user can sign up
+ * Set two factor authentication: 
+   * Disabled: user CAN NOT use two factor authentication
+   * Enforced: users MUST two factor authentication
+   * Enabled: users can decide for themselves if they want to use two factor authentication  
```

Comment:
Left this piece of documentation here because nowhere else is described how the three states of two-factor-authentications work in MOLGENIS